guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -12,38 +12,65 @@ To create a file type repository in a directory on a remote server, see xref:Cre
 
 To create a file type repository in a local directory, complete the following procedure:
 
-. Ensure the Server and {project-client-name} repositories are enabled.
 ifdef::satellite[]
+. Ensure the Server and {project-client-name} repositories are enabled.
+** For {EL} 8:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable={RepoRHEL8BaseOS} \
+--enable={RepoRHEL8AppStream} \
+--enable={RepoRHEL8ServerSatelliteUtils}
+----
+** For {EL} 7:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --enable={RepoRHEL7Server} \
---enable={project-client-RHEL7-url}
+--enable={RepoRHEL7ServerSatelliteUtils}
 ----
 endif::[]
+ifdef::satellite,orcharhino[]
```

Comment:
Why is this limited to satellite/orcharhino?

Reply:
please see the comment above - https://github.com/theforeman/foreman-documentation/pull/1650#discussion_r977769517

Reply:
That doesn't answer the question. Why is this not shown for katello? Or rather, why is this whole `ifdef` even here?

Reply:
By the response of the @maximiliankolb above, I supposed this should be applicable for satellite,orcharhino. So I put the condition there.
Do you suggest to remove that condition completely and make this step of `python3-pulp_manifest` available for all versions?

Reply:
I interpreted what @maximiliankolb said as "please make sure it's also there for orcharhino", but I think it should have been stronger and applied to all builds.

Reply:
ok.